src/App.js

```diff
@@ -1,30 +1,69 @@
-import React, { Fragment, useEffect } from 'react'
+import React, { useState } from 'react'
 import { BrowserRouter, Route, Switch } from 'react-router-dom'
-
+import flatten from 'flat'
 import { Provider } from 'react-redux'
-import store from './store'
+import { store, persistor } from './store'
+import { PersistGate } from 'redux-persist/integration/react'
+
 import SignInPage from './components/pages/SignInPage'
 import SignUpPage from './components/pages/SignUpPage'
 import ProtectedRoutes from './components/routes/ProtectedRoutes'
 import Home from './components/Home'
+import AppLocale from './languageProvider'
```

Comment:
Please change this imports to be absolute imports => https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8